src/marqo/tensor_search/configs.py

```diff
@@ -31,4 +31,5 @@ def default_env_vars() -> dict:
         EnvVars.MARQO_MAX_NUMBER_OF_REPLICAS: 1,
         EnvVars.MARQO_DEFAULT_EF_SEARCH: 2000,
         EnvVars.MARQO_ENABLE_BATCH_APIS: "FALSE",
+        EnvVars.MARQO_MAX_ADD_DOCS_COUNT: 64
```

Comment:
We don't have this in v2. The limit is currently hard-coded

Reply:
removed